projects/storefrontapp/src/main.ts

```diff
@@ -16,6 +16,7 @@ if (environment.production) {
 function bootstrap() {
   platformBrowserDynamic()
     .bootstrapModule(AppModule)
+    /* eslint-disable-next-line no-console */
```

Comment:
Let's add an explanation for the exception:
```suggestion
    /* eslint-disable-next-line no-console 
        --
        It's just an example application file. This message is not crucial
        to be logged using any special logger. Moreover, we don't have
        any special logger available in this context. */
```